index.js

```diff
@@ -150,9 +148,7 @@ instance.prototype.init = function() {
 				self.states['studio_mode'] = false;
 			}
 		});
-		
 		self.obs.on('SceneItemVisibilityChanged', function(data) {
-			if (self.states['studio_mode'] == true) { 
```

Comment:
Was this removal intentional?

Reply:
The removals is not intentional but from what I can see from my side it seams to be tabs thats removed when saving this file with my VSCode.

![image](https://user-images.githubusercontent.com/16779335/101836234-4d3fc200-3b3d-11eb-8ac0-de4299f6f060.png)

Reply:
Something strange is going on... on GitHub it looks like the entire lines were removed. Do you have another commit you needed to push?

Reply:
i can try making a commit from Atom instead of VScode to see if that works better.